Hopsan-release-notes.txt

```diff
@@ -31,6 +31,62 @@
     Version History
 ******************************
 
+2.11.0 (2019-01-??)
+
+--- Enhancements ---
+
+- HopsanCore
+  - Add quantity Resistance with base unit Ohm instead of "ohm".
+    Also adds common unit scales to HopsanGUI.
+  - Add common math functions to embedded script (numhop), see users guide for details.
+    You can now write expression like "x=cos(y)" or "a=atan2(b,c+2)"
+
+- HopsanGenerator:
+  - Added generator for compiler commands that should work with both GCC and Clang.
+    This is only used for library import compilation right now. It will be extended to FMU exporting later.
+    This should enable custom library importing on MacOS from HopsanGUI (but this has not been tested).
+
+- HopsanGUI
```

Comment:
We also made it possible to recompile libraries without closing models where they are used.